bin/compile

```diff
@@ -585,12 +585,25 @@ then
   echo -e "web: $(basename $name)" >> $build/Procfile
 fi
 
+cd $build
 rm -rf $build/.heroku
 
 mkdir -p $build/.profile.d
 echo 'PATH=$PATH:$HOME/bin' > $build/.profile.d/go.sh
 
+if test "${GO_INSTALL_TOOLS_IN_IMAGE}" = "true"
```

Comment:
I tend to like explicit values for things like this as it's the most narrow contract & reduces the chances of accidental activation.